vault.md

```diff
@@ -43,7 +43,55 @@ kubectl -n vault exec vault-0 --  vault kv get kv/hello
 kubectl -n vault exec vault-0 --  vault kv delete kv/hello
 ```
 
-- To uninstall it
+
+### Using vault custom script and vault cli
+
+There is a script in the script folder ready to install and configure vault.
+
+1 - Install vault in the kind cluster.
+
+Go to the scripts folder and launch the script from its location:
+````bash
+./vault.sh
+````
+
+Copy the Root Token shown in the output.
+
+2- Authenticate as Root.
+````bash
+vault login
+````
+Note: paste the Root Token when prompted.
+
+
+3- Create a policy giving access to the path that will stock the secrets
+```bash
+cat <<EOF | vault policy write vault-quickstart-policy -                                                                                                                  ✔  13:07:05
```

Comment:
I suggest to name this policy as `vault-primaza-policy` as the policy will be used with Primaza POC, will be tight a a specific path `kv/primaza`